src/Proto.Actor/EventStream/EventStream.cs

```diff
@@ -20,19 +21,19 @@ public class EventStream : EventStream<object>
     {
         private readonly ILogger _logger = Log.CreateLogger<EventStream>();
 
-        internal EventStream() : this(TimeSpan.Zero, 0)
+        internal EventStream() : this(TimeSpan.Zero, 0, CancellationToken.None)
         {
         }
 
-        internal EventStream(TimeSpan throttleInterval, int throttleCount)
+        internal EventStream(TimeSpan throttleInterval, int throttleCount, CancellationToken ct)
```

Comment:
eventstream takes a CT. in practice it is the killsiwtch CT.

it's used to stop logging deadletter logs once it is triggered for a less noisy shutdown flow